src/sass/navbar.scss

```diff
@@ -0,0 +1,11 @@
+.navbar {
+    .dropdown:hover>.dropdown-menu {
```

Comment:
use & selector in scss

Reply:
https://github.com/ceadoor/cea.ac.in/pull/31/commits/4eb4d3ed13f8dfbe4fb44ac857dd51c2a58c56f9